pg/benchmarks/pg_scope_all.rb

```diff
@@ -0,0 +1,23 @@
+require 'bundler/setup'
+require 'pg'
+require_relative 'support/benchmark_pg'
+
+# Connect to POSTGRES database with provided values or use default
```

Comment:
I'll prefer if we leave out the comment here. I believe the code is self explanatory.